datafusion/physical-expr/src/utils.rs

```diff
@@ -679,6 +679,19 @@ pub fn reassign_predicate_columns(
     })
 }
 
+/// Reverses the ORDER BY expression, which is useful during equivalent window
+/// expression construction. For instance, 'ORDER BY a ASC, NULLS LAST' turns into
+/// 'ORDER BY a DESC, NULLS FIRST'.
+pub fn reverse_order_bys(order_bys: &[PhysicalSortExpr]) -> Vec<PhysicalSortExpr> {
```

Comment:
`reverse_order_bys` is moved under `sort_expr.rs` to be able to use it in `src/physical_plan/aggregates/mod.rs`. With this change scope of the function is increased.